### PR TITLE
spike: add privacy benchmark

### PR DESCRIPTION
**Pull request by taccatisid**
## PR description

Add a benchmark job based on dockerized caliper. 

## Changelog

- [ ] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

**Comment by atoulme** [comment]
Please fix the url, it doesn’t look like this ties to a public resource?

File: benchmark/config/caliper-benchconfig-offchain.yaml~ [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#
```
Comment (macfarla) [inline benchmark/config/caliper-benchconfig-offchain.yaml~:1]: should this file be deleted?

File: benchmark/config/#caliper-benchconfig-offchain.yaml# [hunk 2]
```diff
@@ -0,0 +1,80 @@
+#
```
Comment (macfarla) [inline benchmark/config/#caliper-benchconfig-offchain.yaml#:1]: should this file be deleted

File: benchmark/config/caliper-networkconfig-offchain.json~ [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (macfarla) [inline benchmark/config/caliper-networkconfig-offchain.json~:1]: should this file be deleted

**Review by macfarla — APPROVED** [review]
LGTM

**Pull request merged**